composer.json

```diff
@@ -25,20 +25,20 @@
         "php":                      "^7.0",
         "phpspec/prophecy":         "^1.5",
         "phpspec/php-diff":         "^1.0.0",
-        "sebastian/exporter":       "^1.0 || ^2.0",
-        "symfony/console":          "^2.7 || ^3.0",
-        "symfony/event-dispatcher": "^2.7 || ^3.0",
-        "symfony/process":          "^2.7 || ^3.0",
-        "symfony/finder":           "^2.7 || ^3.0",
-        "symfony/yaml":             "^2.7 || ^3.0",
+        "sebastian/exporter":       "^2.0",
+        "symfony/console":          "^3.2",
```

Comment:
Ah, now I see that PR target branch is 4.0. Probably by the time phpspec 4 will be released symfony 2.7 will be close to its end of life.
Now this makes sense for me, thanks for the explanation.